tests/coloring/Test_Graph.cpp

```diff
@@ -0,0 +1,105 @@
+#include <gtest/gtest.h>
+
+#include <graph.h>
+
+namespace UT
```

Comment:
Why we need additional namespace in tests?

Reply:
Because there can be multiple types of tests, this namespace covers `unit` tests